.buildkite/pipeline_pr.py

```diff
@@ -85,16 +95,22 @@ def group(group_name, command, agent_tags=None, priority=0, timeout=30):
     agent_tags=["ag=1"],
 )
 
-pipeline = {
-    "agents": {"queue": "default"},
-    "steps": [
-        step_style,
+steps = [step_style]
+
+changed_files = get_changed_files("main")
+if any(x.suffix != ".md" for x in changed_files):
```

Comment:
If we do this change, we also need to remove the markdown lint integration test (which I would be in favor of), because now if some markdown file contains a space at the end of some line, it will make the _next_ PR that doesn't touch documentation fail.